packages/rainbowkit/src/wallets/walletConnectors/enkryptWallet/enkryptWallet.ts

```diff
@@ -59,21 +59,21 @@ export const enkryptWallet = ({
             steps: [
               {
                 description:
-                  'We recommend pinning Enkrypt Wallet to your taskbar for quicker access to your wallet.',
+                  'wallet_connectors.extension.enkrypt.step1.description',
                 step: 'install',
-                title: 'Install the Enkrypt Wallet extension',
+                title: 'wallet_connectors.extension.enkrypt.step1.title',
               },
               {
                 description:
-                  'Be sure to back up your wallet using a secure method. Never share your secret phrase with anyone.',
+                  'wallet_connectors.extension.enkrypt.step2.description',
                 step: 'create',
-                title: 'Create or Import a Wallet',
+                title: 'wallet_connectors.extension.enkrypt.step2.title',
```

Comment:
Instead of importin `i18n` and then doing `i18n.t()` we will specify these instructions as key strings. So when `connectorsForWallets` is getting executed we will take those keys and use `i18n` in `connectorsForWallets` 🙏